server/src/main/scala/za/co/absa/atum/server/api/database/Runs.scala

```diff
@@ -35,33 +37,34 @@ class Runs (implicit dBEngine: SlickPgEngine) extends DBSchema{
 
 object Runs {
 
-  private def scalaIterableToSQLArray(toConvert: Seq[Any]): String = {
-    SerializationUtils.asJson(toConvert)
-      .replace("[", "{")
-      .replace("]", "}")
+  private def nestedScalaSeqToPgArray[T <: AnyRef: ClassTag](toConvert: Seq[Seq[T]]): String = {
+    val scalaSeqJsonized = toConvert.map(scalaSeqToPgArray).mkString(",")
+    "{" + scalaSeqJsonized + "}"
+  }
+
+  private def scalaSeqToPgArray[T <: AnyRef: ClassTag](toConvert: Seq[T]): String = {
```

Comment:
Why the type limitation (`<: AnyRef: ClassTag`)?

Reply:
This is the definition of `asJson` function that I'm calling inside: `def asJson[T <: AnyRef](obj: T): String` and this generic type `T` here is also limited, so the 'outer' `T` here in my function cannot be a superset of it. The build was failing